lib/active_merchant/billing/gateways/visanet_peru.rb

```diff
@@ -77,7 +77,7 @@ def scrub(transcript)
         transcript.
           gsub(%r((Authorization: Basic )\w+), '\1[FILTERED]').
           gsub(%r((\"cardNumber\\\":\\\")\d+), '\1[FILTERED]').
-          gsub(%r((\"cvv2Code\\\":)\d+), '\1[FILTERED]')
+          gsub(%r((\"cvv2Code\\\":\\\")\d+), '\1[FILTERED]')
```

Comment:
Ahhh..  Good catch @shasum!  10 out of the 11 remote tests for this gateway are failing for me because I don't have credentials. Sorry!

Thanks for the fix!

Any chance we can add test credentials to our fixtures file here?  https://github.com/activemerchant/active_merchant/blob/master/test/fixtures.yml#L1015